src/site/site.xml

```diff
@@ -63,6 +63,9 @@
 
         <links>
             <item
+                name="Home"
+                href="http://james.apache.org/index.html" />
```

Comment:
href is wanted to be a link to the Marketting website
(not a review remark, just explaination)